From 2fd9f851511d9312889810cb1fa8eae2468a2a75 Mon Sep 17 00:00:00 2001
From: ningyu <ningyu@eswincomputing.com>
Date: Thu, 7 Aug 2025 14:56:04 +0800
Subject: [PATCH 448/533] fix:multi define gpio cause d314 boot fail

Changelogs:
Add cpu voltage adjust gpio94, and delete gpio default define in pinctrl within d314

Signed-off-by: ningyu <ningyu@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-d314.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-d314.dts b/arch/riscv/boot/dts/eswin/eic7700-d314.dts
index 0c05768360f4..07331a6867cb 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-d314.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-d314.dts
@@ -319,8 +319,7 @@ &pinctrl_gpio37_default &pinctrl_gpio38_default &pinctrl_gpio39_default &pinctrl
 			&pinctrl_gpio41_default &pinctrl_gpio46_default &pinctrl_gpio52_default
 			&pinctrl_gpio53_default &pinctrl_gpio64_default &pinctrl_gpio65_default &pinctrl_gpio66_default
 			&pinctrl_gpio67_default &pinctrl_gpio70_default &pinctrl_gpio73_default &pinctrl_gpio83_default
-			&pinctrl_gpio86_default &pinctrl_gpio87_default &pinctrl_gpio92_default &pinctrl_gpio93_default
-			&pinctrl_gpio94_default>;
+			&pinctrl_gpio86_default &pinctrl_gpio87_default &pinctrl_gpio92_default &pinctrl_gpio93_default>;
 
     /* pin header default function for GPIO
        SPI1 (CS0,SCLK,MOSI,MISO,D2,D3): GPIO 35,36,37,38,39,40
-- 
2.52.0

